tests/test_blscache.py

```diff
@@ -0,0 +1,58 @@
+from chia_rs import G1Element, PrivateKey, AugSchemeMPL, G2Element, BLSCache
+from chia.util.ints import uint64
+from chia.types.blockchain_format.sized_bytes import bytes48
+import pytest
+from typing import List
+
+
+def test_instantiation() -> None:
```

Comment:
would it make sense to have tests, in python, that validate that the rust implementation behaves exactly like the python one?